prove/Develop03/Scripture.cs

```diff
@@ -25,7 +27,7 @@ public void Display()
             Console.WriteLine($"{_reference}:");
             foreach (Word word in _words)
             {
-                if (word.CheckIfHidden())
+                if (word.IsHidden())
```

Comment:
This should be handled by the Word class.  The Word has all the information it needs to hide itself.  Don't make the Scripture class handle this part.